examples/text-feature-extraction/README.md

```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Feature Extraction Examples
+
```

Comment:
@pi314ever, another thing - we usually add the link for the original script that was adapted for Gaudi2 in the code file (run_feature_extraction) and the README. If you wrote the feature extraction sample entirely yourself, this can be ignored